spec/helpers/helpers.rb

```diff
@@ -0,0 +1,14 @@
+module Helpers
+  def log_in_as(user, options = {})
+    options[:password]
+    options[:email]
+    session[:user_id] = user.id
+  end
+
+  def feature_log_in_as(user)
```

Comment:
And if you remove the other method, you can just call this method: `login_as(user)` which is shorter and more succinct :)

Reply:
Done in sha: 5e2fd92